schema/me/conversations.js

```diff
@@ -39,52 +39,54 @@ export const BuyerOutcomeTypes = new GraphQLEnumType({
   },
 });
 
+export const ConversationFields = {
+  id: {
+    description: 'Impulse id.',
+    type: GraphQLString,
+  },
+  inquiry_id: {
+    description: 'Gravity inquiry id.',
+    type: GraphQLString,
+  },
+  from_id: {
+    type: new GraphQLNonNull(GraphQLString),
+  },
+  from_type: {
+    type: new GraphQLNonNull(GraphQLString),
+  },
+  from_name: {
+    type: new GraphQLNonNull(GraphQLString),
+  },
+  from_email: {
+    type: new GraphQLNonNull(GraphQLString),
+  },
+  to_id: {
+    type: new GraphQLNonNull(GraphQLString),
+  },
+  to_type: {
+    type: new GraphQLNonNull(GraphQLString),
+  },
+  to_name: {
+    type: new GraphQLNonNull(GraphQLString),
+  },
+  buyer_outcome: {
+    type: GraphQLString,
```

Comment:
Should we go ahead and change the type to `BuyerOutcomeType` or is that something different?